home_audit.node.js

```diff
@@ -1157,23 +1205,52 @@ let validationRules = {
     },
 
     /*
-     * Gets wall length for window area validations
+     * Gets the first wall dimension for window area validations
      */
-    _get_wall_length: function() {
+    _get_wall_dimension_1: function() {
```

Comment:
It's not clear which of the `_get_wall_dimension_*` functions is for the sides and which for the front/back. I would name them in a way that makes that more clear.